.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,24 @@
+name: CIFuzz
+on: [pull_request]
+jobs:
+  Fuzzing:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Build Fuzzers
+      id: build
+      uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
```

Comment:
Should we be using the master branch?

Reply:
Default branch is master so ya.

Reply:
link : https://google.github.io/oss-fuzz/getting-started/continuous-integration/#branches-and-paths

Reply:
>       uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master

is a docker path